composer.json

```diff
@@ -1,18 +1,32 @@
 {
     "name": "phphants/phph-site",
     "description": "PHP Hampshire website",
+    "type": "project",
+    "homepage": "https://phphants.co.uk",
     "license": "MIT",
-    "keywords": [
-        "php",
-        "hampshire"
-    ],
-    "homepage": "http://www.phphants.co.uk/",
     "require": {
-        "php": ">= 5.4",
-        "zendframework/zendframework": "2.*",
-        "sabre/vobject": "3.*"
+        "php": "^7.0",
+        "roave/security-advisories": "dev-master",
+        "zendframework/zend-expressive": "^1.0",
+        "zendframework/zend-expressive-helpers": "^2.0",
+        "zendframework/zend-stdlib": "^3.0",
+        "zendframework/zend-expressive-fastroute": "^1.2",
+        "zendframework/zend-servicemanager": "^3.1",
+        "zendframework/zend-expressive-zendviewrenderer": "^1.1",
+        "sabre/vobject": "^4.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.2.*"
+        "phpunit/phpunit": "^5.5",
+        "squizlabs/php_codesniffer": "^2.6"
+    },
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/App/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "AppTest\\": "test/AppTest/"
+        }
     }
```

Comment:
Perhaps add some scripts for running tests and phpcs? 

```
{
    "scripts": {
        "phpcs": "phpcs",
        "test": "phpunit"
    }
}
```

So that rather than running `./vendor/bin/phpcs` it's `composer phpcs`